AccountingSystem/Client/AccountingCLI.py

```diff
@@ -0,0 +1,258 @@
+"""
+AccountingCLI class implementing command line administrative interface to
+DIRAC Accounting DataStore Service
+"""
+
+__RCSID__ = "$Id$"
+
+import sys
+
+from DIRAC.Core.Base.CLI import CLI, colorize
+from DIRAC.Core.Utilities import ExitCallback
+from DIRAC.Core.DISET.RPCClient import RPCClient
+from DIRAC import gLogger
+
+class AccountingCLI( CLI ):
+
+  def __init__( self ):
+    CLI.__init__( self )
+    self.do_connect( None )
+    self.identSpace = 20
```

Comment:
identSpace is used everywhere
but it should be called indentSpace, because it is the space for indentation, right?